ion/tokenizer.go

```diff
@@ -356,9 +356,9 @@ func (t *tokenizer) ReadValue(tok token) (string, error) {
 	case tokenSymbolOperator, tokenDot:
 		str, err = t.readOperator()
 	case tokenString:
-		str, err = t.readString()
+		str, err = t.readString(false)
```

Comment:
Open question: This PR adds an unlabeled boolean to several methods; is there a way we can make the meaning function call parameters like this more obvious?

Is it worth making `readClobString`, `readLongClobString`, etc? I suspect the answer is no, but you have more context here than I do.

Reply:
> is there a way we can make the meaning function call parameters like this more obvious?

How about:
```
const isClob = true
const isNotClob = false
```
And using these two where I need to distinguish clobs from string:   ```str, err = t.readString(isNotClob)```


> Is it worth making readClobString, readLongClobString, etc?

I think the majority of what is being done in these functions are similar between clob and string, except some small changes.

Reply:
> ```go
> const isClob = true
> const isNotClob = false
> ```

This is a good approach. My only quibble with it is that `isClob` and `isNotClob` sound like open-ended boolean values rather than constants with well-known values. For example, if I read this:

```go
str, err = t.readString(isClob)
```

it seems like I'll need to track down the definition of `isClob` to see whether it's `true` or `false`. How about:

```go
const clobText = true;
const nonClobText = false;
```

The downside is you have booleans that don't "look like" booleans (they don't start with `is`), but the upside is that the function calls end up being more self-documenting:

```go
str, err = t.readString(clobText);
// or
str, err = t.readString(nonClobText);
```

Thoughts?

Reply:
Yeah, I think ```clobText``` and ```nonClobText``` are better options, specially given we will name our input variable ```isClob```.

Will update the PR.